docs/self-managed/setup/deploy/local/c8run.md

```diff
@@ -60,13 +60,13 @@ All Camunda 8 Run components can be accessed using the username/password combina
 
 Tasklist and Operate are available at:
 
-- Tasklist: `http://localhost:8080/tasklist`
-- Operate: `http://localhost:8080/operate`
+- Tasklist: [http://localhost:8080/tasklist](http://localhost:8080/tasklist)
+- Operate: [http://localhost:8080/operate](http://localhost:8080/operate)
 
 The following components do not have a web interface, but the URLs may be required for additional configuration:
 
-- Zeebe Gateway: `http://localhost:26500`
-- Connectors: `http://localhost:8085`
+- Zeebe Gateway: [http://localhost:26500](http://localhost:26500)
+- Connectors: [http://localhost:8085](http://localhost:8085)
```

Comment:
This formatting doesn't _feel_ like it makes sense for the non-web interface component URLs, but I understand the desire to be consistent. Can you bring this to the next tech writer sync?